spec/cocina/models/validators/description_types_validator_spec.rb

```diff
@@ -441,4 +442,28 @@
       end.to raise_error(Cocina::Models::ValidationError)
     end
   end
+
+  describe 'when an invalid Description with string keys' do
+    let(:props) do
+      {
+        'title' => [{ 'value' => 'The Structure of Scientific Revolutions' }],
+        'purl' => 'https://purl.stanford.edu/bc123df4567',
+        'contributor' => [
+          {
+            'name' => [
+              {
+                'value' => 'Kuhn, Thomas'
+              }
+            ],
+            'type' => 'foo',
+            'status' => 'primary'
+          }
+        ]
+      }
+    end
+
+    it 'is not valid' do
+      expect { validate }.to raise_error(Cocina::Models::ValidationError)
```

Comment:
Am I reading this right that `DescriptionTypesValidator.validate` will throw a validation exception if given attributes that have string keys, but that you can still instantiate an instance of `DescriptionTypesValidator` with attributes that have string keys?

Reply:
`DescriptionTypesValidator.validate` will properly validate if given props regardless of whether keys are strings or symbols.